src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -22,6 +22,17 @@ const DRUPAL_CACHE_FILENAME = 'drupal/pages.json';
 // should pull the latest Drupal data.
 const PULL_DRUPAL_BUILD_ARG = 'pull-drupal';
 
+// We need to pull the Drupal content if we have --pull-drupal OR if
+// the content is not available in the cache.
+const shouldPullDrupal = buildOptions => {
```

Comment:
I extracted this logic for reuse in [`setUpFeatureFlags`](https://github.com/department-of-veterans-affairs/vets-website/pull/10586/files#diff-b18d5a20aff9790323e66b853e6c290fR159).